head_ref/src/HeadReference.cpp

```diff
@@ -35,9 +35,9 @@ HeadReference::HeadReference() :
     n.param<std::string>("tf_prefix", tf_prefix_, "");
     n.param<std::string>("pan_joint_name", pan_joint_props_.name, "neck_pan_joint");
     n.param<std::string>("tilt_joint_name", tilt_joint_props_.name, "neck_tilt_joint");
-    n.param<std::string>("frame_mount", frame_mount_, "head_mount");
-    n.param<std::string>("frame_neck", frame_neck_, "neck_tilt");
-    n.param<std::string>("frame_head", frame_head_, "top_kinect/openni_camera");
+//    n.param<std::string>("frame_mount", frame_mount_, "head_mount");
```

Comment:
I'm sorry for being nitpicky, but it's not considered good practice to keep commented code. In a while, no-one will know the purpose anymore. Could you remove this?